external/vcm/vcm/derived_mapping.py

```diff
@@ -175,3 +175,19 @@ def is_sea(self):
 def is_sea_ice(self):
     # one hot encoding for sea ice surface
     return xr.where(vcm.xarray_utils.isclose(self["land_sea_mask"], 2), 1.0, 0.0)
+
+
+@DerivedMapping.register("Q1", required_inputs=["dQ1", "pQ1"])
```

Comment:
Good point. I also changed the policy and docstring to include required variables even in the not-well-defined case (e.g. e.g. `dQu` only needs `dQxwind, dQywind` if `dQu` is not already in the mapping, but `dQxwind, dQywind` should still be listed as `required_inputs`. This is because the usage of this registry is for when an output is explicitly requested as a derived output variable and thus it is assumed the variable does not already exist and needs to be derived.